experimental/dds/tree2/src/feature-libraries/modular-schema/fieldKind.ts

```diff
@@ -84,6 +84,7 @@ export class FieldKindWithEditor<
 	public constructor(
 		identifier: TName,
 		multiplicity: TMultiplicity,
+		// TODO: Should this be unknown instead?
```

Comment:
Why would FieldChangeHandler be unknown?

Reply:
I meant `FieldChangeHandler<unknown, TEditor>`. I think the `any` here ends up leading to some bad type checking somewhere else. I just remembered that I ran into this a long time ago, discussed it with Craig, and came up with some plan to address it, but we never got around to it.

Reply:
Makes sense. Thanks for clarifying.